test/workflow.vader

```diff
@@ -752,44 +760,68 @@ Execute (On update):
 Execute (When already installed):
   call plug#begin()
   Plug 'junegunn/vim-easy-align', { 'do': 'touch installed2' }
-  Plug 'junegunn/vim-pseudocl', { 'do': 'touch installed2' }
```

Comment:
FYI, I specified two plugins here to make vim-plug start parallel installer. Now we have more plugins, we can safely remove it, which you did, nice.